### DISCUSSION TITLE
How to make buttons with captions using Python?

### DISCUSSION THREAD

**Discussion by LordZirconium** [Rapptz/discord.py#6535] [Q&A] [answered]
How to make buttons with captions using Python?
Help me please!
Join my project on the link !!! - [Project](https://code-with-me.jetbrains.com/xL63gS3L6XF4udPGRec4cw#p=PC&fp=DD81188F60F0253D9E2F42178E6D5F2AABD214A4EF23C7547B86D2E867D560EB) 
I will be glad to everything!
_**Thank!**_

**Comment by 15696** [comment, marked as answer]
I don't know what you mean. Also this is the discord.py repository, may not be what you are looking for.

**Reply by LordZirconium** [reply]
Probably I'm confusing something. Sorry!
